entrypoint.sh

```diff
@@ -0,0 +1 @@
+poetry run gunicorn "todo_app.app:create_app()" --bind 0.0.0.0:${PORT:-5000}
```

Comment:
It's a good habit to always add a [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) to your scripts;
```suggestion
#!/bin/bash
poetry run gunicorn "todo_app.app:create_app()" --bind 0.0.0.0:${PORT:-5000}
```